tests/test_core.py

```diff
@@ -277,6 +287,6 @@ def test_tablenames(self):
             self.assertEqual(SqliteDict.get_tablenames(fname), ['table1'])
         with SqliteDict(fname,tablename='table2') as db2:
             self.assertEqual(SqliteDict.get_tablenames(fname), ['table1','table2'])
-        
```

Comment:
@mpenkov can we set up an automated git hook to check for these things?

Code with trailing whitespace should never have been merged.